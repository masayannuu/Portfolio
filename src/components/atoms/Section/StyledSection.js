import styled from 'styled-components'
import { Section } from 'reactbulma'

const StyledSection = styled(Section)`
  @media screen and (max-width: 480px) {
    width: 100%;
    margin-bottom: 150px;
  }
  @media screen and (min-width: 481px) {
    width: 90%;
    margin-left: auto;
    margin-right: auto;
    margin-bottom: 229px;
  }
`

export default StyledSection
