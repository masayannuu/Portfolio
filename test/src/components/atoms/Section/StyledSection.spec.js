import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import StyledSection from '../../../../../src/components/atoms/Section/StyledSection'

describe('Section StyledSection Atoms', () => {
  it('should render StyledSection', () => {
    const wrapper = Enzyme.shallow(<StyledSection>StyledSection</StyledSection>)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
