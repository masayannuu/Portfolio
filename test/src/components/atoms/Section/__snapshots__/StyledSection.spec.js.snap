// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section StyledSection Atoms should render StyledSection 1`] = `
<Section
  className="StyledSection-bqFfPK fZFPUU"
>
  StyledSection
</Section>
`;
